experimental/packages/opentelemetry-instrumentation-grpc/test/helper.ts

```diff
@@ -1006,6 +1010,11 @@ export const runTests = (
             'rpc.response.metadata.server_metadata_key':
```

Comment:
The test description should be updated to include `client and server spans` to accommodate the new changes.

Reply:
Changed. Thank you 👌